src/api/axios-client/index.ts

```diff
@@ -0,0 +1,82 @@
+import axios, {
+	type AxiosInstance,
+	type AxiosRequestConfig,
+	type AxiosResponse,
+} from "axios";
+
+import {
+	HttpClient,
+	type HttpRequestConfig,
+	type HttpResponse,
+} from "../http-client";
+import { HttpError } from "../http-client/error";
+
+//TODO: 서버와 맞추기
+const BASE_URL = "";
+
+class AxiosClient extends HttpClient {
+	private readonly client: AxiosInstance;
+
+	constructor() {
+		super();
+		const axiosInstance = axios.create({
+			baseURL: BASE_URL,
+			withCredentials: true,
+		});
+		this.client = axiosInstance;
+	}
+
+	async request<
+		ResponseBodyT = unknown,
```

Comment:
default가 unknown이지않나욤 ?

Reply:
아 abstract request 메서드에서 지정해줘서 여기선 안써줘도 될것 같긴하네요.   
여기서 명시해주지 않으면 하나 더 들어가보기 전엔 any로 인지돼서 넣은 것 같은데, 굳이.. 싶다면 제거하는게 나으려나요?

Reply:
아 그것보다는 제네릭 타입의 기본은 unknown으로 추론되기때문에 unknown을 기본으로 지정해주지않아도 될거라는 이야기였어요
`ResponseBodyT = unknown` -> `ResponseBodyT`
```suggestion
		ResponseBodyT,
```